src/js/url-helpers.js

```diff
@@ -0,0 +1,22 @@
+export function getQueryParam(paramName) {
```

Comment:
Nice to see a small focussed, file. We need to break more of `script.js` into something like this.